### DISCUSSION TITLE
Upgrading from @keystone-next to @keystone-6

### DISCUSSION THREAD

**Discussion by traviscode0004** [keystonejs/keystone#8497] [Technical Discussion]
I started a project a few years ago using keystone-next and I am revisiting the project and looking for directions to migrate from keystone-next to keystone6.

**Comment by dcousens** [comment]
@traviscode0004 this will be difficult to answer, as `@keystone-next` had a high number of breaking changes throughout it's lifetime, and was not officially supported.

If you attempt the upgrade, you could post your experience and problems in this discussion as a way to share with others, and maybe we can help too.

**Reply by traviscode0004** [reply]
If there is no documentation for the upgrade then I will roll with what I have. Everything is working fine. If I change my mind I will make an update here.
